### ISSUE TITLE
A box must be specified

### ISSUE THREAD

**Issue by sasos90** [mitchellh/vagrant#7114]
Hy everyone!

I have two questions about vagrant errors.

1. My old vagrant box suddenly stopped working, so I made a vagrant update and so on.. My "vagrant up" is not working anymore either. I recreated the archive so I have the latest archive from puphpet page and I also updated the vagrant "Vagrant 1.8.1", but I just keep getting the error: "vm: * A box must be specified."

  Before I had problems with mounting, that's why I installed the vbguest plugin for vagrant, and it installed it correctly (after some issues again ofcourse..) but after that I am getting some errors again like "Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported." So now I really don't know what to do more. I can not install a vagrant box anymore.

2. Why Vagrant is never stable? I think I could run it perfectly only once.. I always had some troubles there.

**Comment by sethvargo** [comment]
Hi @sasos90 

Thank you for opening an issue, and I am sorry you are experiencing issues with Vagrant. Unfortunately there is not enough information to reproduce the issues you are experiencing. I'm very sorry you are having so much trouble with Vagrant - that is certainly never our intention. Are you able to complete the attached issue template so we can try to reproduce your issue? https://raw.githubusercontent.com/mitchellh/vagrant/master/.github/ISSUE_TEMPLATE.md

Thanks!

**Comment by sasos90** [comment]
Thanks for quick reply!

My data:
Vagrant version = 1.8.1
My local OS = Linux Mint
Guest operating system: Ubuntu 14.04 (Trusty 64bit)
Vagrantfile: http://laravel.io/bin/REMEe#

I have no debug output, because suddenly it just started to work, although I got some SSH error at the end. But I was able to "vagrant ssh" and it obviously symlink the folders correctly.
But since I have some problems with graphics, my computer die every now and then, so  then I need to "vagrant up" again. So if I will get the error that time, I will let you know.
Anyway please try to run it and tell me if it works for you.

Thanks!

**Comment by sethvargo** [comment]
Hi @sasos90 

Unfortunately without a reproduction case, it is very difficult for our core team to address this issue. If it happens again, please complete the issue template that shows when you create a new issue and complete all the fields. This will help us in identifying and fixing the issue in the future. Thanks! :smile:
